docs/source/guide/migrationpy3.rst

```diff
@@ -1,12 +1,24 @@
 .. _guide_migration_py3:
 
-Migrating from Python 2.7 to Python 3
-=====================================
-Python 2.7 was deprecated by the `Python Software Foundation <https://www.python.org/psf-landing/>`_ back on January 1, 2020 following a multi-year process of phasing it out. Because of this, AWS has deprecated support for Python 2.7, so versions of boto3 and botocore released after the deprecation date will no longer work with Python 2.7.
+Migrating to Python 3
+=====================
+
+Python 2.7 was deprecated by the `Python Software Foundation <https://www.python.org/psf-landing/>`_
+on January 1, 2020 following a multi-year process of phasing it out. Because of this, AWS has
+deprecated support for Python 2.7, which means that releases of Boto3 issued after the deprecation
+date will no longer work on Python 2.7.
+
+This affects both modules that comprise the AWS SDK for Python: Botocore (the underlying low-level
+module) and Boto3 (which implements the API functionality and higher-level features).
 
 Timeline
 --------
-Going forward, all projects using the AWS SDK for Python need to transition to using Python 3, with Python 3.6 becoming the minimum by the end of the transition. The deprecation dates for the affected versions of Python are:
+Going forward, all projects using Boto3 need to transition to using Python 3, with
+Python 3.6 becoming the minimum by the end of the transition. Boto3 and Botocore will end support
+for Python 2.7 effective July 15, 2021, while Python 3.5 and lower will need to be updated to Python
```

Comment:
At this point we may as well yank the 3.4/3.5 reference since it's already passed and it doesn't require migrating to Python 3.

Reply:
In which case, the table becomes a one-row table, which seems pointless and it probably needs to come back out. Your thoughts, @nateprewitt?